src/main/java/io/stargate/sgv2/jsonapi/config/constants/VectorConstant.java

```diff
@@ -13,17 +13,17 @@ public interface VectorConstant {
       Map.of(
           "ada002",
           SimilarityFunction.DOT_PRODUCT,
-          "openai_v3_small",
+          "openai-v3-small",
           SimilarityFunction.DOT_PRODUCT,
-          "openai_v3_large",
+          "openai-v3-large",
           SimilarityFunction.DOT_PRODUCT,
           "bert",
           SimilarityFunction.DOT_PRODUCT,
           "gecko",
           SimilarityFunction.DOT_PRODUCT,
-          "nv_qa_4",
+          "nv-qa-4",
           SimilarityFunction.DOT_PRODUCT,
-          "cohere_v3",
+          "cohere-v3",
           SimilarityFunction.DOT_PRODUCT,
           "other",
           SimilarityFunction.COSINE);
```

Comment:
According to the [document](https://docs.datastax.com/en/astra-db-serverless/cql/query-vector-data-with-cql.html#create-the-vector-schema), here should be hyphens not underscores. Since changing the value in here will affect the code in tables. I will create another PR to replace those hardcoded values to Emun.

Reply:
move `SUPPORTED_SOURCE_MODELS` map from `VectorConstants` to `SourceModel`